benchmark/tpch/create_table_s3.sh

```diff
@@ -0,0 +1,151 @@
+echo "CREATE TABLE IF NOT EXISTS nation
+(
+    n_nationkey  INTEGER not null,
+    n_name       STRING not null,
+    n_regionkey  INTEGER not null,
+    n_comment    STRING
+)" | $MYSQL_CLIENT_CONNECT
+
+echo "CREATE TABLE IF NOT EXISTS region
+(
+    r_regionkey  INTEGER not null,
+    r_name       STRING not null,
+    r_comment    STRING
+)" | $MYSQL_CLIENT_CONNECT
+
+echo "CREATE TABLE IF NOT EXISTS part
+(
+    p_partkey     BIGINT not null,
+    p_name        STRING not null,
+    p_mfgr        STRING not null,
+    p_brand       STRING not null,
+    p_type        STRING not null,
+    p_size        INTEGER not null,
+    p_container   STRING not null,
+    p_retailprice DOUBLE not null,
+    p_comment     STRING not null
+)" | $MYSQL_CLIENT_CONNECT
+
+echo "CREATE TABLE IF NOT EXISTS supplier
+(
+    s_suppkey     BIGINT not null,
+    s_name        STRING not null,
+    s_address     STRING not null,
+    s_nationkey   INTEGER not null,
+    s_phone       STRING not null,
+    s_acctbal     DOUBLE not null,
+    s_comment     STRING not null
+)" | $MYSQL_CLIENT_CONNECT
+
+echo "CREATE TABLE IF NOT EXISTS partsupp
+(
+    ps_partkey     BIGINT not null,
+    ps_suppkey     BIGINT not null,
+    ps_availqty    BIGINT not null,
+    ps_supplycost  DOUBLE  not null,
+    ps_comment     STRING not null
+)" | $MYSQL_CLIENT_CONNECT
+
+echo "CREATE TABLE IF NOT EXISTS customer
+(
+    c_custkey     BIGINT not null,
+    c_name        STRING not null,
+    c_address     STRING not null,
+    c_nationkey   INTEGER not null,
+    c_phone       STRING not null,
+    c_acctbal     DOUBLE   not null,
+    c_mktsegment  STRING not null,
+    c_comment     STRING not null
+)" | $MYSQL_CLIENT_CONNECT
+
+echo "CREATE TABLE IF NOT EXISTS orders
+(
+    o_orderkey       BIGINT not null,
+    o_custkey        BIGINT not null,
+    o_orderstatus    STRING not null,
+    o_totalprice     DOUBLE not null,
+    o_orderdate      DATE not null,
+    o_orderpriority  STRING not null,
+    o_clerk          STRING not null,
+    o_shippriority   INTEGER not null,
+    o_comment        STRING not null
+)" | $MYSQL_CLIENT_CONNECT
+
+echo "CREATE TABLE IF NOT EXISTS lineitem
+(
+    l_orderkey    BIGINT not null,
+    l_partkey     BIGINT not null,
+    l_suppkey     BIGINT not null,
+    l_linenumber  BIGINT not null,
+    l_quantity    DOUBLE not null,
+    l_extendedprice  DOUBLE not null,
+    l_discount    DOUBLE not null,
+    l_tax         DOUBLE not null,
+    l_returnflag  STRING not null,
+    l_linestatus  STRING not null,
+    l_shipdate    DATE not null,
+    l_commitdate  DATE not null,
+    l_receiptdate DATE not null,
+    l_shipinstruct STRING not null,
+    l_shipmode     STRING not null,
+    l_comment      STRING not null
+)" | $MYSQL_CLIENT_CONNECT
+
+# copy data to s3 bucket
+
+# create external stage
+echo"
+create stage tpch_data url='s3://...'
+  connection = (
+        ENDPOINT_URL = 'https://s3.amazonaws.com'
+        ACCESS_KEY_ID = '...'
```

Comment:
Could be change to env vars?
ref: https://github.com/datafuselabs/databend/blob/main/scripts/benchmark/query/load/hits.sh#L124